lib/linter/linter.js

```diff
@@ -1340,6 +1340,7 @@ class Linter {
                 providedOptions.physicalFilename
             );
         } catch (err) {
+            err.messageForTest = err.message; // Original exception message without extra helper text for asserting in fatal test cases.
```

Comment:
I'm not sure how to hit this codepath in the tests. This is in `_verifyWithoutProcessors()`.

Reply:
I would assume creating a rule that throws an error would hit this?